app/importers/reading/somerville_mega_reading_importer.rb

```diff
@@ -47,10 +48,11 @@ def reset_counters!
     # TODO set instance variables to 0
   end
 
-  def read_or_fetch_csv
+  def read_or_fetch_sheet
     raise 'unfinished'
-    # TODO read from google or mock and return `streaming_csv` that responds
-    # to #each_with_index
+    # fetcher = GoogleSheetsFetcher.new
+    # fetcher.get_spreadsheet(SHEET_URL)
+    # TODO add sheet url to ENV
```

Comment:
I'm figuring you left this so I can add in the env bits when shipping, thanks!  If there's something else to know about here just let me know.  I think this will be folder-based, like this: https://github.com/studentinsights/studentinsights/blob/master/app/importers/transitions/bedford_davis_transition_notes_importer.rb#L58

Reply:
Can you also either comment on what you're thinking about for the other methods here that still `raise 'unfinished'?  I'm figuring that some of them will still need to be done in the next pass (eg, doing the "syncing" with existing records in the database) but that others are now stale and can be removed.